app/script/auth/config.js

```diff
@@ -16,37 +16,8 @@
  * along with this program. If not, see http://www.gnu.org/licenses/.
  *
  */
+import * as Environment from './Environment';
 
-export const SUPPORTED_LANGUAGE = [
-  'cs',
-  'da',
-  'de',
-  'el',
-  'es',
-  'et',
-  'fi',
-  'fr',
-  'hr',
-  'hu',
-  'it',
-  'lt',
-  'nl',
-  'pl',
-  'pt',
-  'ro',
-  'ru',
-  'sk',
-  'sl',
-  'tr',
-  'uk',
-];
-export const DEFAULT_LANGUAGE = 'en';
-
-export function getLocale() {
-  return navigator.languages && navigator.languages.length ? navigator.languages[0] : navigator.language;
-}
-
-export function currentLanguage() {
-  const LANGUAGE_SHORTHAND_LENGTH = 2;
-  return getLocale().substr(0, LANGUAGE_SHORTHAND_LENGTH) || DEFAULT_LANGUAGE;
-}
+export const APP_ENVIRONMENT = window.APP_ENVIRONMENT || Environment.PRODUCTION;
```

Comment:
No need to define `|| Environment.PRODUCTION` because it's already the default of `getEnvironment()` from `Environment.js`

Reply:
I changed it inside `Environment `

Reply:
👍